### PR TITLE
15 - Creating an Order API with the required return

### PR DESCRIPTION
**Pull request by aMotherFox**

### CODE REVIEW

File: java/FoodTruckApp/pom.xml [hunk 1]
```diff
@@ -44,6 +44,11 @@
 			<artifactId>spring-boot-starter-jdbc</artifactId>
 		</dependency>
 <!--		added in postgres^-->
+		<dependency>
+			<groupId>org.modelmapper</groupId>
+			<artifactId>modelmapper</artifactId>
+			<version>3.1.0</version>
+		</dependency>
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp/pom.xml]: It doesn't seem like you are using this anywhere, so you probably don't need this dependency, right?

Comment (aMotherFox) [reply java/FoodTruckApp/pom.xml]: That's right, thank you, it has been removed

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/AppetizerOrdered.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.my.FoodTruckApp.order;
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/AppetizerOrdered.java]: I think this belongs in the  appetizer directory

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/EntreeOrdered.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.my.FoodTruckApp.order;
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/EntreeOrdered.java]: this should probably be in the entree dir

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderController.java [hunk 4]
```diff
@@ -1,59 +1,36 @@
...
-    @GetMapping("/orders/{id}")
-    public Order getOrderById(@PathVariable Integer id) {
-        return orderService.getOrderById(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
-    }
-    //------------------------create order-----------------------------------------------------------------------------------------------------------
+//    @GetMapping("/orders/{id}")
+//    public Order getOrderById(@PathVariable Integer id) {
+//        return orderService.getOrderById(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
+//    }
     @PostMapping("/orders")
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderController.java]: do you need this? If so, can this be official documentation?

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderService.java [hunk 5]
```diff
@@ -1,101 +1,52 @@
...
-//        ArrayList<Entree> entrees = entreeRepository.getAllEntrees();
-//        System.out.println("These are the entrees: " + entrees);
-//        return entrees;
-//    }
     public ArrayList<Order> getListOfOrders() {
-        ArrayList<Order> orders = orderRepository.getAllOrders();
-        System.out.println("There are the orders: " + orders);
+        ArrayList<Order> orders = orderRepository.getListOfOrders();
         return orders;
     }
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderService.java]: Do you wanna return the result right the way, or do you need to do something else with it first?

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/AppetizerOrdered.java (hunk 2 above)
Comment (aMotherFox) [reply java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/AppetizerOrdered.java]: This has been moved

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/EntreeOrdered.java (hunk 3 above)
Comment (aMotherFox) [reply java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/EntreeOrdered.java]: This has been moved

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderController.java (hunk 4 above)
Comment (aMotherFox) [reply java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderController.java]: I do not, it has been removed

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderService.java (hunk 5 above)
Comment (aMotherFox) [reply java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderService.java]: I do not need to do anything with it, it has been inlined

File: java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/entree/EntreeOrdered.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class EntreeOrdered {
+    private Integer orderId;
+    private Integer entreeId;
+}
+//when creating models named after a table, name it the same thing
```
Comment (LucasCostaAtCyberSaint) [inline java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/entree/EntreeOrdered.java]: is there a better place for this note/doc?

Comment (aMotherFox) [reply java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/entree/EntreeOrdered.java]: It has been moved to a notes doc

**Review by LucasCostaAtCyberSaint — APPROVED** [review]

**Pull request merged**